### ISSUE TITLE
Text decompile bug

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/asdec#44]
```
What steps will reproduce the problem?
1. editing texts with sothink swf decompiler
2. saving file
3. opening file in FFdec

What is the expected output? What do you see instead?
at 49% "Can't open SWF file"

What version of the product are you using? On what operating system?
FFdec 1.4.1, newest version, Windows XP Service Pack 3

Could you provide the SWF file you have problem with?
If the answer is yes, then please attach it here or send me it via email.
Could you at least attach PCode source?
Last edited codes: ui/profile/MyFriends.as line 181
file: http://www.mediafire.com/?8y5o6p6tac8c1i6
=============================
mar 14, 2013 6:32:53 EM com.jpexs.decompiler.flash.Main$OpenFileWorker 
doInBackground
SEVERE: null
com.jpexs.decompiler.flash.abc.NotSameException: Streams are not the same at 
pos 000000de
        at com.jpexs.decompiler.flash.abc.CopyOutputStream.write(CopyOutputStream.java:66)
        at com.jpexs.decompiler.flash.SWFOutputStream.write(SWFOutputStream.java:77)
        at com.jpexs.decompiler.flash.SWFOutputStream.writeUB(SWFOutputStream.java:302)
        at com.jpexs.decompiler.flash.SWFOutputStream.writeSB(SWFOutputStream.java:321)
        at com.jpexs.decompiler.flash.SWFOutputStream.writeRECT(SWFOutputStream.java:354)
        at com.jpexs.decompiler.flash.tags.DefineFont2Tag.getData(DefineFont2Tag.java:128)
        at com.jpexs.decompiler.flash.SWFInputStream.readTag(SWFInputStream.java:1112)
        at com.jpexs.decompiler.flash.SWFInputStream.readTagList(SWFInputStream.java:853)
        at com.jpexs.decompiler.flash.SWF.<init>(SWF.java:248)
        at com.jpexs.decompiler.flash.Main.parseSWF(Main.java:184)
        at com.jpexs.decompiler.flash.Main$OpenFileWorker.doInBackground(Main.java:211)
        at javax.swing.SwingWorker$1.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at javax.swing.SwingWorker.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
===============================================

Do you have the original source code which produced the wrong
decompilation? If yes, then please attach it.
No, the problem was after editing the text, not before

Please provide any additional information below.
/
```

Original issue reported on code.google.com by `mk14...@gmail.com` on 14 Mar 2013 at 5:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 19 Mar 2013 at 7:35
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hi,
it is fixed in version 1.4.2, try it
```

Original comment by `jindra.p...@gmail.com` on 24 Mar 2013 at 11:53
- Changed state: **Fixed**
